.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,19 @@
+## Description
+
+<!-- Explain what this Pull Request changes -->
+
+<!-- Link any issues if applicable.
+Use keywords from https://docs.github.com/en/get-started/writing-on-github/working-with-advanced-formatting/using-keywords-in-issues-and-pull-requests
+-->
+
+<!-- Uncomment based on the type of your changes below -->
+
+<!--
+## This is a **Content Change**
+- [ ] Changes have been tested.
```

Comment:
why would I test that tho lol

Reply:
https://github.com/eludris/elusite/pull/2/commits/5f483daff42f5e566890794bbd4dab0e411695b1